pkgs/applications/misc/lifelines/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchgit, autoconf, libtool, automake, pkgconfig, gettext, libiconv
+, bison, ncurses, perl }:
+
+stdenv.mkDerivation rec {
+  version = "2017-06-12-unstable";
+  name = "lifelines-${version}";
+
+  src = fetchgit {
```

Comment:
GitHub also provides autogenerated archives for arbitrary revs, which is cheaper than a checkout (and caches better). Use `fetchFromGitHub` instead of `fetchgit` to easily make use of this.